run-ib-igprof

```diff
@@ -47,8 +47,35 @@ for prof in ${PROFILES} ; do
       done
       cd -
     else
-      runTheMatrix.py $WF --command " -n $EVENTS --profile $prof" > ./runTheMatrix.log 2>&1 || ERR=1
-      mv runTheMatrix.log $(ls -d $WORKFLOW*)
+      scram tool info tensorflow
+      case $SCRAM_ARCH in
+              el8*)
+              scram setup /cvmfs/cms-ib.cern.ch/sw/x86_64/latest/$SCRAM_ARCH/cms/cmssw/CMSSW_15_1_MKLDNN0_*/config/toolbox/$SCRAM_ARCH/tools/selected/tensorflow.xml;;
```

Comment:
this might fail is there are non or more than one `MKLDNN0_` IBs

Reply:
I updated the script to use the latest MKLDNN0 IB or set ERR=1 if the scram setup command fails.